lightyear/src/transport/wrapper/mod.rs

```diff
@@ -0,0 +1,15 @@
+//! Module defining 'wrappers' that modify the behaviour of an existing [`PacketReceiver`] or [`PacketSender`].
+//!
+//! Wrappers are used to add additional functionality to an existing transport, such as encryption, compression, metrics, etc.
```

Comment:
A lot of http servers call this "middleware", which could be an alternative name.

Reply:
yes i like that!